src/widgets/buttonmatrix/lv_buttonmatrix.h

```diff
@@ -67,6 +68,14 @@ typedef bool (*lv_buttonmatrix_button_draw_cb_t)(lv_obj_t * btnm, uint32_t btn_i
 
 LV_ATTRIBUTE_EXTERN_DATA extern const lv_obj_class_t lv_buttonmatrix_class;
 
+#if LV_USE_OBJ_PROPERTY
+enum {
```

Comment:
The docs CI is failing because of the anonymous enums.

```suggestion
enum _lv_property_buttonmatrix_id_t {
```